src/Components/CommentCard/CommentCard.jsx

```diff
@@ -67,6 +73,25 @@ const CommentCard = ({ comment }) => {
         <div className=" text-xl mr-8" style={{ color: themeObject.text }}>
           {comment?.votes?.downvotedBy?.length}
         </div>
+        <span
+          className="material-symbols-rounded  ml-auto mr-8"
+          onClick={() =>
+            editComment(comment?.postId, comment?._id, authToken)
+          }
+          style={{ color: themeObject.text }}
+        >
+          edit
+        </span> */}
```

Comment:
**suggestion**: delete unwanted code if not needed

Reply:
Thanks for reminding this.